src/Extension/CoreExtension.php

```diff
@@ -1479,7 +1479,13 @@ function twig_get_attribute(Environment $env, Source $source, $object, $item, ar
                 return true;
             }
 
-            return $object[$arrayItem];
+            if ($type === 'method') {
+                if (is_callable($object[$arrayItem])) {
+                    return $object[$arrayItem](...$arguments);
```

Comment:
This is not properly integrated with the sandbox system.

Reply:
Done!